src/app/(authentication)/forgot-password/page.tsx

```diff
@@ -0,0 +1,349 @@
+          variant="h6"
+          color="blue-gray"
+          className={` -mb-3`}>
+          Mã xác thực <span className="text-red-700 font-bold text-lg">*</span>
+        </Typography>
+
+        <div className="flex relative w-full max-w-[24rem]">
+          <Input
+            size="lg"
+            className=" !border-t-blue-gray-200 focus:!border-t-gray-900 pr-20"
+            labelProps={{
+              className: 'before:content-none after:content-none',
+            }}
+            {...register('resetCode', {
+              required: 'Vui lòng nhập mã xác thực',
+            })}
+            containerProps={{
+              className: 'min-w-0',
+            }}
+            crossOrigin={undefined}
+          />
+
+          {isCodeExpired ? (
+            <Button
+              placeholder={undefined}
+              size="sm"
+              color="blue"
+              variant="text"
+              className="!absolute right-1 top-1.5 rounded"
+              onClick={resendCode}>
+              Gửi lại
+            </Button>
+          ) : (
+            <Countdown
+              key={timer.current}
+              date={timer.current + EMAIL_ACTIVATION_CODE_TIMER}
+              renderer={countdownRenderer}
+              onComplete={() => setIsCodeExpired(true)}
+            />
+          )}
+        </div>
+        <ErrorInput
+          // This is the error message
+          errors={errors?.resetCode?.message}
+        />
+      </div>
+
+      <div className="mb-1 flex flex-col gap-6">
+        <Typography
+          placeholder={undefined}
+          variant="h6"
+          color="blue-gray"
+          className={` -mb-3`}>
+          Mật khẩu mới <span className="text-red-700 font-bold text-lg">*</span>
+        </Typography>
+
+        <div className="flex relative w-full max-w-[24rem]">
+          <Input
+            size="lg"
+            type="password"
+            className=" !border-t-blue-gray-200 focus:!border-t-gray-900 pr-20"
+            labelProps={{
+              className: 'before:content-none after:content-none',
+            }}
+            {...register('newPassword', {
+              required: 'Vui lòng nhập mật khẩu mới',
+            })}
+            containerProps={{
+              className: 'min-w-0',
+            }}
+            crossOrigin={undefined}
+          />
+        </div>
+        <ErrorInput
+          // This is the error message
+          errors={errors?.newPassword?.message}
+        />
+      </div>
+
+      <Button
+        type="submit"
+        placeholder={undefined}
+        ripple={true}
+        className={`mt-[3rem] w-full bg-blue-800 text-white rounded-md py-4`}>
+        Đổi mật khẩu
+      </Button>
+
+      <Button
+        onClick={handleBack}
+        placeholder={undefined}
+        ripple={true}
+        className={`mt-[1rem] w-full bg-blue-800 text-white rounded-md py-4`}>
+        Trở lại
+      </Button>
+    </form>
+  )
+}
+
+function SignupForm() {
```

Comment:
```suggestion
function SignupForm() {
```
đổi tên lại cho đúng ngữ cảnh